Gems/ROS2/Code/Include/ROS2/Sensor/Events/EventSourceAdapter.h

```diff
@@ -0,0 +1,192 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <ROS2/ROS2Bus.h>
+#include <ROS2/Sensor/Events/SensorEventSource.h>
+#include <ROS2/Sensor/SensorConfiguration.h>
+
+namespace ROS2
+{
+    namespace Internal
+    {
+        template<
```

Comment:
maybe turn off clang format here.

Reply:
We agreed, through discussion, that this formatting is acceptable here.